kvstoremesh/option/config.go

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package option
+
+import (
+	"github.com/spf13/pflag"
+
+	"github.com/cilium/cilium/pkg/option"
+)
+
+const (
+	// PprofAddress is the default value for pprof in kvstoremesh
+	PprofAddress = "localhost"
+
+	// PprofPort is the default value for pprof in kvstoremesh
+	PprofPort = 6064
```

Comment:
Would be nice to document this in the following table along with the other pprof ports:

https://github.com/cilium/cilium/blob/68b3ae1918b1bd81e2f82d4fb83c896f8fae4f9f/Documentation/operations/system_requirements.rst?plain=1#L413-L436